ksp_plugin_adapter/source_location.cs

```diff
@@ -4,10 +4,14 @@
 
 namespace System.Runtime.CompilerServices {
 
+#if KSP_VERSION_1_7_3
+
```

Comment:
What happens to those in 1.8.1?

Reply:
See the comments above:
```C#
// Pretend that we have the caller file path and line number attributes, even
// though they do not exist in .NET 3.5; since this is actually substituted
// at compile time it will work as intended.
```

They are in the .NET Framework 4.5 and later, so they are there; we don't need to pretend that they are.